main.js

```diff
@@ -1,3 +1,29 @@
+const menuList = document.querySelector('.menuList');
+const menuAdd = document.querySelector('.menuAdd');
+const menuContact = document.querySelector('.menuContact');
+const sectionList = document.querySelector('.sectionList');
+const newBook = document.querySelector('.newBook');
```

Comment:
- [ ] Good job so far with your awesome book app :clap: When I add a book, I can't get my list automatically I have to close the tab and restart the page again before getting my app works as expected. Please fix it in order to give a better user experience and make your app works as expected :+1: 
![Screenshot from 2022-12-01 18-13-25](https://user-images.githubusercontent.com/6615394/205131562-b6c5cdb8-5512-4569-aa9d-5d8745f71ee8.png)